lib/plugins/slack/attachements.js

```diff
@@ -93,10 +93,20 @@ module.exports = {
       let color = 'danger';
```

Comment:
This should be 'good', in case the metric isn't available (e.g. default of coachScore when not running coach)